app/Models/City.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Models;
+
+use Database\Factories\CityFactory;
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Illuminate\Database\Eloquent\Model;
+
+class City extends Model
+{
+    protected $fillable = ['name'];
```

Comment:
Here you could use `protected $guarded = ['id'];`  instead of fillable.
That way you don't have to add every attribute you want to add to your model to the fillable array. Its easier to keep updated the guarded array.